src/components/v5/shared/Button/Button.tsx

```diff
@@ -112,6 +113,9 @@ const Button = forwardRef<HTMLButtonElement, PropsWithChildren<ButtonProps>>(
             >
               {children}
             </ButtonContent>
+            {showNotificationDot && (
```

Comment:
what if we add a new prop called `rightContent` or something like that and we just compose this dot in from where we need to? And we can also pull it into a reusable component called `NotificationDot`

Reply:
Realised I can just include it in the {children} so I have removed this!